// // import React from 'react'

// // function Errorboundry32({userName}) {

// //     if (userName === "joker")
// //     {
// //         throw new Error("Not a hero")
// //     }
// //   return (
// //     <div>{userName}</div>
// //   )
// // }

// // export default Errorboundry32



// /////////////////////////////////////////
// App.js file 



// import './App.css';
// import DemoRef from './Components/DemoRef';
// import Errorboundry32 from './Components/Errorboundry32';
// import EventBind from './Components/EventBind';
// import Form from './Components/Form';
// import Fragment from './Components/Fragment';
// import InlineCss from './Components/InlineCss';
// import InputFocus from './Components/InputFocus';
// import NameList from './Components/NameList';
// import ParntCom from './Components/ParntCom';



// function App() {
//   return (
//     <div className="App">

//  <Errorboundry32 userName ="Rahul" />
//  <Errorboundry32 userName ="Dev" />

//  <Errorboundry32 userName ="Dev" />
//     </div>
//   );
// }

// export default App;


// Error boundry  only stop those code which have  error other code run perfacly.